documentation/data-flow.md

```diff
@@ -0,0 +1,61 @@
+React London Data Flow
+======================
+
+React London is a mostly static isomorphic React site.
+
+Rather than having its own database it consumes a GraphQL API exposed by the
+[Badger Brain][brain] application, which in turn gets its data from a managed
+third party content management system called [Prismic][prismic].
+
+[brain]: https://github.com/redbadger/badger-brain
+[prismic]: https://prismic.io
+
+The data flow looks like this:
+
+
+    Prismic -> BadgerBrain -> React.London
+
+
+## Updating content
+
+If we want to update the content of the site we log into Prismic, search for
+the appropriate documentent (the React.London community and the contained
+events, etc), make our edits, and then save and publish the documents.
+
+React.London queries BadgerBrain for site data every time a user visits the
+site, so the user will always get the latest content in Prismic.
+
+Andy is spearheading the creation of a content editor's guide. Once more
+complete we shall link it, until then ask him to see it.
```

Comment:
Is it [this](https://andrewbestbier1.gitbooks.io/red-badger-prismic-guide/content/)?